README.md

```diff
@@ -1 +1,186 @@
+  <ul>
+    <li><a href="https://www.w3schools.com/html/default.asp">Html</a></li>
+    <li><a href="https://www.w3schools.com/css/default.asp">CSS</a></li>
+    <li><a href="https://www.w3schools.com/js/">Javascript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> This project has the following features
+
+- Create /src file for source files using webpack
+- Used /dist directory to share webpack
+- Separate the code into different modules
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!--video description-->
+<!-- LIVE DEMO -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+Open terminal on the same folder of the project and run:
+
+```sh
+ npm install
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/brhanuhailu/to-Do-list.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Module2-Awsome-Books
+  npm install
+```
+
+### Usage
+
+To run the project, open index.html
+
+### Run tests
+
+To run tests, run the following command:
+
+To check for html errors run:
+
+```sh
+  npx hint .
+```
+
+To check for css errors run:
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+
+To check for js errors run:
+
+```sh
+  npx eslint .
+```
+
+### Deployment
+
+Deployed using Github Pages.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Brhanu Hailu**
+
+- GitHub: [@brhanuhailu](https://github.com/brhanuhailu)
+- Twitter: [@tigraycountry](https://twitter.com/TigrayCountry)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/brhanu-hailu-85578a246/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Add popup.
+
+- [ ] Contact details
+- [ ] Frameworks
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/brhanuhailu/to-Do-list.git/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> we are so excited to accept you feedback to improve my profession.
+
+If you like this project like it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+we would like to thank Microverse organization
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT]https://github.com/brhanuhailu/to-Do-list.git) licensed.
+(Check the LICENSE file)
```

Comment:
- [ ] you did a great job 😃 at adding the MIT LICENCE, but I see that when clicking on the link to open the page of LICENCE I see that it is redirecting me to the page of your todo page in GitHub, not to the licence page, so please 😊 try to put the correct link for the MIT LICENCE to let anyone when clicking on the link of LICENCE redirect him/her to the page of the LICENCE 💯